Upgrade to Angular 8

bampakoa opened the pull request:
@stevermeister I have upgraded the library to Angular 8. Could you please have a look? There are also some points that need to be discussed:

1. I have noticed that we reference `@angular-devkit/build-ng-packagr` and `tsickle` in the `package.json`. Do we use them outside of the CLI?
2. With the advent of Angular 8, differential loading came along. Do you think turning this on will benefit our users?

stevermeister commented on projects/ngx-wig/src/lib/ngx-wig.component.ts:
```diff
@@ -56,7 +56,7 @@ export class NgxWigComponent implements AfterViewInit,
   public contentChange = new EventEmitter();
 
-  @ViewChild('ngWigEditable', { read: ElementRef })
+  @ViewChild('ngWigEditable', { read: ElementRef, static: true })
```
👍

stevermeister commented:
@bampakoa that's great! 

> Do we use them outside of the CLI?
I think that was automatically generated by packager

> differential loading
sure! 

Also, I'm thinking about to sync the version of ngx-wig with Angular, at least a major one.

bampakoa commented:
@stevermeister I updated also the `ng-packagr` related dependencies and I added support for differential loading. BTW I think that differential loading is not so much applicable to libraries in contrast to applications.

stevermeister commented:
do you know why tests keep failing? 
I see strange error about `yarn `

bampakoa commented:
I have detected the following from the output of Travis:

> error @angular-devkit/build-angular@0.803.17: The engine "node" is incompatible with this module. Expected version ">= 10.9.0". Got "9.2.1"
> error Found incompatible module
> info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

Are you able to execute `yarn` locally?

stevermeister commented:
> Are you able to execute yarn locally?

it was possible with only warning:


```
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
```

stevermeister commented:
yes, found 
>  The engine "node" is incompatible with this module. Expected version ">= 10.9.0".

bampakoa commented:
@stevermeister this is ready

stevermeister commented:
cool, thx!

stevermeister commented:
published version8 - https://www.npmjs.com/package/ngx-wig
not sure whether we need to keep in sync minor versions, probably it's nice to start at least with major versions

bampakoa commented:
No, I think that we should not stick with Angular minor versions. Even Angular CLI and Angular Components are not aligned with Angular minor 😉 . Though a note in README file explaining why we jumped to version 8 would be a good idea 😃

stevermeister commented:
good point, updated
